UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1844,10 +2046,9 @@ public void OpenCodeFile(string name, CodeEditorMode editorDecompile)
             {
                 Focus();
 
-                CodeEditorDecompile = editorDecompile;
+                //TODO: find the way to scroll to the code item and highlight it.
```

Comment:
Oh, I meant to not just remove them, but revert them, so it should be like before:
_(replaces lines from 2049 through 2051)_
```
CodeEditorDecompile = editorDecompile;

HighlightObject(code);
ChangeSelection(code);
```

Reply:
Ah, sorry misunderstood what you were saying.